experiments/standalone/Bucket/compare_gpu_cpu_output.jl

```diff
@@ -0,0 +1,9 @@
+using DelimitedFiles
+using Statistics
+import ClimaLSM
+outdir = joinpath(pkgdir(ClimaLSM), "experiments/standalone/Bucket/artifacts")
+cpu_state = readdlm(joinpath(outdir, "tf_state_cpu.txt"), ',')
+gpu_state = readdlm(joinpath(outdir, "tf_state_gpu.txt"), ',')
+@show mean(cpu_state .- gpu_state)
+@show eps(Float64)
```

Comment:
do we want to print `eps(Float64)` here? I think it would get printed if the assertion fails which is maybe all we need (not a big deal either way though)

Reply:
It actually wasnt printing it! (also, apologies for all the failed buildkite runs. no GPU locally so i am trial and error debugging this using buildkite. not very efficient, I should have just worked on the cluster!)

and, I dont know what a good threshold is. the value of the error is 6e-15. eps(Float64) is ~2e-16. The mean error will be lower than the individual value error, so perhaps this is artificially low, but in any case it seems like good agreement.

Reply:
I am using 100*eps(Float64) now.

Reply:
I think 100*eps(Float64) or 50*eps(Float64) (~1e-14) would be good!